pkg/api/unversioned/time_proto.go

```diff
@@ -22,37 +22,23 @@ import (
 	"time"
 )
 
-// ProtoTime is a struct that is equivalent to Time, but intended for
+// Timestamp is a struct that is equivalent to Time, but intended for
 // protobuf marshalling/unmarshalling. It is generated into a serialization
 // that matches Time. Do not use in Go structs.
-type ProtoTime struct {
-	// Represents the time of an event.
-	Timestamp Timestamp `json:"timestamp"`
-}
-
-// Timestamp is a protobuf Timestamp compatible representation of time.Time
 type Timestamp struct {
 	// Represents seconds of UTC time since Unix epoch
 	// 1970-01-01T00:00:00Z. Must be from from 0001-01-01T00:00:00Z to
 	// 9999-12-31T23:59:59Z inclusive.
-	Seconds int64 `json:"seconds"`
-	// Non-negative fractions of a second at nanosecond resolution. Negative
-	// second values with fractions must still have non-negative nanos values
-	// that count forward in time. Must be from 0 to 999,999,999
-	// inclusive.
-	Nanos int32 `json:"nanos"`
+	Seconds int64 `json:"seconds" protobuf:"varint,1,opt,name=seconds"`
```

Comment:
We don't support nanos on any of our time fields today (they're zeroed) but I can add them back here.  We'd need to set our expectations on time which we've never done.